cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ScalarFunctionsTest.scala

```diff
@@ -87,6 +88,42 @@ class ScalarFunctionsTest extends DocumentingTest {
         resultTable()
       }
     }
+    section("elementId()", "functions-element-id") {
+      p("""The function `elementId()` returns a node or relationship identifier, unique within a specific transaction and DBMS.
+        """.stripMargin('#'))
+      note {
+        p("""Guarantees:
+            #
+            #* Every node and relationship has an element id.
+            #* The identifier is unique among other nodes' and relationships' identifiers within the scope of a single transaction and DBMS.
+            #  In other words, element id is also unique across different databases/graphs in the same DBMS.
+            #
+            #Example of properties that are NOT guaranteed:
+            #
+            #* No guarantees are given regarding the order of the returned id values.
+            #* No guarantees are given about the length of the id string values.
+            #* Outside of the scope of a single transaction, no guarantees are given about the mapping between id values and elements.
+          """.stripMargin('#'))
```

Comment:
Suggestions for language improvements are very welcome, had a hard time expressing this. Thanks!